### PR TITLE
Create integration test for vector unmatched size(insert/find)

### PR DESCRIPTION
**Pull request by Yuqi-Du**
**What this PR does**: Create integration tests for vector unmatched size(insert/find)

**Which issue(s) this PR fixes**:
Fixes #<497>

**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java [hunk 1]
```diff
@@ -1315,6 +1316,123 @@ public void deleteOne() {
...
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(jsonVectorStrCount3)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, vectorSizeTestCollectionName)
+          .then()
+          .statusCode(200)
+          .body("errors", is(notNullValue()))
+          .body("errors[0].message", endsWith("Expected vector of 5 size, but received 3"));
```
Comment (vkarpov15) [inline src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java:1350]: maybe better to say "expected vector of length 5", "5 size" reads a bit awkwardly.

**Review by vkarpov15 — APPROVED** [review]
One minor comment, LGTM otherwise

**Review by vkarpov15 — APPROVED** [review]

**Review by jeffreyscarpenter — APPROVED** [review]

File: src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java (hunk 1 above)
Comment (Yuqi-Du) [reply src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java]: Thanks! Fixed that.

**Comment by Yuqi-Du** [comment]
Merged, two IT failed, that is because they rely on a new release for Main Repo. 
[Main Repo change, merged not released](https://github.com/stargate/stargate/pull/2726)

**Pull request merged**